src/raft.rs

```diff
@@ -2893,4 +3036,36 @@ impl<T: Storage> Raft<T> {
             pr.ins.set_cap(cap);
         }
     }
+
+    /// Whether this RawNode is active recently.
+    pub fn is_recent_active(&self, id: u64) -> bool {
+        if let Some(pr) = self.prs().get(id) {
+            if pr.recent_active {
```

Comment:
You mean `self.prs().get(id).map_or(false, |pr| pr.recent_active)`?

Reply:
Yes, I will modify it later.